### PR TITLE
Namespace Sidekiq::Cron by Rails environment

### PR DESCRIPTION
**Pull request by berfarah**
@swapdisc This work?

### CODE REVIEW

File: lib/tasks/sidekiq_cron/load.rake [hunk 1]
```diff
@@ -3,10 +3,9 @@ namespace :sidekiq_cron do
   task load: :environment do
     config = Sidekiq::Cron::Tasks.config
 
-    next unless File.exist?(config.file)
+    next if context.hash.empty?
```
Comment (swapdisc) [inline lib/tasks/sidekiq_cron/load.rake]: Is this method provided by the `Rake` library? I could only locate it in irb, and calling `empty?` on it there would throw an error.

Comment (swapdisc) [inline lib/tasks/sidekiq_cron/load.rake]: (`context`, I mean)

Comment (berfarah) [inline lib/tasks/sidekiq_cron/load.rake]: Was supposed to be config 🤦‍♂️

File: lib/tasks/sidekiq_cron/load.rake [hunk 2]
```diff
@@ -3,10 +3,9 @@ namespace :sidekiq_cron do
   task load: :environment do
     config = Sidekiq::Cron::Tasks.config
 
-    next unless File.exist?(config.file)
+    next if config.schedule.empty?
```
Comment (swapdisc) [inline lib/tasks/sidekiq_cron/load.rake]: The guard clause shouldn't be necessary - but why not? 🤷‍♂️

Comment (berfarah) [inline lib/tasks/sidekiq_cron/load.rake]: It handles the scenario where for whatever reason the config file couldn't be found. That said, it also prevents deleting everything. Tradeoffs

**Pull request merged**